### PR TITLE
[CI]:  Updated command used for updating error codes docs

### LINKED ISSUE #78: [CI] update command used for updating error codes docs
Currently in the [err-ref-publisher](https://github.com/layer5io/meshkit/blob/master/.github/workflows/error-ref-publisher.yaml) workflow, we update the docs using:
https://github.com/layer5io/meshkit/blob/8577bc689c39420e58e39967abd1f4957d002d9a/.github/workflows/error-ref-publisher.yaml#L49
This might fail in some conditions.
We should use this instead:
```
$ echo '{ "errors_export": "" }' | jq --slurpfile export ./helpers/errorutil_errors_export.json '.errors_export = $export[0]' > ./meshery/docs/_data/errorref/meshkit_errors_export.json

```

### PR DESCRIPTION
**Pull request by includestdlibrary**
**Description**
Updated command used for updating error codes docs

This PR fixes #78 

**Notes for Reviewers**

**[Signed commits](../CONTRIBUTING.md#signing-off-on-commits-developer-certificate-of-origin)**
- [x] Yes, I signed my commits.

### CODE REVIEW

File: .github/workflows/error-ref-publisher.yaml [hunk 1]
```diff
@@ -46,7 +46,7 @@ jobs:
 
       - name: Update and push docs
         run: |
-          echo '{ "errors_export": '$(cat ./helpers/errorutil_errors_export.json)' }' | jq > ./meshery/docs/_data/errorref/meshkit_errors_export.json
+          $ echo '{ "errors_export": "" }' | jq --slurpfile export ./helpers/errorutil_errors_export.json '.errors_export = $export[0]' > ./meshery/docs/_data/errorref/meshkit_errors_export.json
```
Comment (DelusionalOptimist) [inline .github/workflows/error-ref-publisher.yaml:49]: Thanks @includestdlibrary. Just a very minor change and we'll be good to go : D
```suggestion
          echo '{ "errors_export": "" }' | jq --slurpfile export ./helpers/errorutil_errors_export.json '.errors_export = $export[0]' > ./meshery/docs/_data/errorref/meshkit_errors_export.json
```

Comment (includestdlibrary) [reply .github/workflows/error-ref-publisher.yaml:49]: Ohh yess sorry 😅😅 I have not deleted that

**Comment by includestdlibrary** [comment]
@DelusionalOptimist  please review now

File: .github/workflows/error-ref-publisher.yaml (hunk 1 above)
Comment (DelusionalOptimist) [reply .github/workflows/error-ref-publisher.yaml]: Gotcha :grin:

**Review by DelusionalOptimist — APPROVED** [review]
LGTM! :rocket:

**Comment by leecalcote** [comment]
Good deal.

**Pull request merged**